root/genre/List.js

```diff
@@ -32,6 +32,13 @@ const GenreList = ({genres}: PropsT) => (
           </li>
         ))}
       </ul>
+      <p>
+        {l('Is a genre missing from the list? Request it by')} + 
```

Comment:
The paragraph should not be split into several strings, as you can assert the localization in other languages will follow the same schema. Please use only one call to `l` instead. There are several examples in the code for embedding links into a localizable string.

Reply:
@yvanzo  I was able to find some examples which I used to emulate the syntax. :)